src/rgw/driver/rados/rgw_notify.cc

```diff
@@ -1201,25 +1216,30 @@ int publish_commit(rgw::sal::Object* obj,
       event_entry.retry_sleep_duration = topic.cfg.dest.retry_sleep_duration;
       bufferlist bl;
       encode(event_entry, bl);
-      const auto& queue_name = topic.cfg.dest.persistent_queue;
+      const std::string bucket_name = res.bucket->get_name(); 
+      const std::string object_key = res.object_name ? *res.object_name : "";
```

Comment:
i think this is redundant ? am i wrong to assume that an event will always have an object associated with it so object name will `res.object_name` will never be `nullptr`